kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStore.scala

```diff
@@ -46,10 +46,17 @@ class JDBCMetadataStore(conf: KyuubiConf) extends MetadataStore with Logging {
 
   private val dbType = DatabaseType.withName(conf.get(METADATA_STORE_JDBC_DATABASE_TYPE))
   private val driverClassOpt = conf.get(METADATA_STORE_JDBC_DRIVER)
+  private lazy val mysqlDriverClass =
+    try {
+      Class.forName("com.mysql.cj.jdbc.Driver", false, Thread.currentThread.getContextClassLoader)
+      "com.mysql.cj.jdbc.Driver"
+    } catch {
+      case _: ClassNotFoundException => "com.mysql.jdbc.Driver"
+    }
```

Comment:
I just realized there is a method `ReflectUtils#isClassLoadable`, that can simplify the code.